packages/roosterjs-editor-api/lib/format/getBlockQuoteState.ts

```diff
@@ -0,0 +1,32 @@
+import { Editor } from 'roosterjs-editor-core';
+import { NodeType } from 'roosterjs-editor-types';
+import { getTagOfNode } from 'roosterjs-editor-dom';
+
+// Crawl up the DOM tree from given start node to find if it falls on a blockquote
```

Comment:
I don't think we need this function since getBlockQuoteElement() can already do the same job.

Reply:
Removed this function.